wifiphisher/pywifiphisher.py

```diff
@@ -187,7 +187,8 @@ def shutdown(template=None, network_manager=None):
         template.remove_extra_files()
 
     print '[' + R + '!' + W + '] Closing'
-    sys.exit(0)
+    for c in phishinghttp.creds:
+        sys.exit('[' + R + '+' + W + '] Captured credentials:' + R + c)
```

Comment:
Note that the `shutdown()` method also runs when the `-qS` option is not enabled. In that case, the user may have obtained credentials from different victims that would appear as different list elements.